tests/phpunit/tests/blocks/renderCommentTemplate.php

```diff
@@ -512,4 +512,97 @@ public function test_rendering_comment_template_unmoderated_preview() {
 			'Should not include any unapproved comments after removing filter'
 		);
 	}
+
+	public function test_rendering_comment_template_sets_comment_id_context() {
```

Comment:
`@ticket` and `@covers` where appropriate (I know the trac ticket is TBD! 😄 )

Reply:
https://github.com/WordPress/wordpress-develop/pull/4507/commits/c6257da39005ea91363287b4507669b7432c6688

Reply:
3ad0fb7bd8